src/OrbitGl/TimeGraph.h

```diff
@@ -56,6 +56,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   void DrawText(float layer);
 
   void RequestUpdatePrimitives();
+  void RequestUpdate() override { RequestUpdatePrimitives(); }
```

Comment:
Should we completely remove `RequestUpdatePrimitives` and replace it with `RequestUpdate` for consistency?

Reply:
I will replace it.